### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Tauras-Rim**
-Added ChartService, SuggestionsService, OrderList tests
-Some code adjustments

### CODE REVIEW

File: Energy Saver/Services/SuggestionsService.cs [hunk 1]
```diff
@@ -13,10 +13,13 @@ public List<decimal> GetLatestTaxComparison(List<List<Taxes>> taxes)
 
             if(list.Count > 1)
             {
-                comparison.Add(Math.Round(list.First().GasAmount * 100 / list.ElementAt(1).GasAmount - 100));
-                comparison.Add(Math.Round(list.First().ElectricityAmount * 100 / list.ElementAt(1).ElectricityAmount - 100));
-                comparison.Add(Math.Round(list.First().WaterAmount * 100 / list.ElementAt(1).WaterAmount - 100));
-                comparison.Add(Math.Round(list.First().HeatingAmount * 100 / list.ElementAt(1).HeatingAmount - 100));
+                if(list.ElementAt(1).GasAmount != 0 && list.ElementAt(1).ElectricityAmount != 0 && list.ElementAt(1).WaterAmount != 0 && list.ElementAt(1).HeatingAmount != 0)
```
Comment (Leminode) [inline Energy Saver/Services/SuggestionsService.cs:16]: list may contain 0 elements

File: Energy Saver/Model/Serialization.cs [hunk 2]
```diff
@@ -7,56 +7,6 @@ namespace Energy_Saver.Model
 {
     public static class Serialization
```
Comment (priolette) [inline Energy Saver/Model/Serialization.cs]: Would've been great if this class was renamed to something like "Utilities"

File: Energy Saver/Services/TableService.cs [hunk 3]
```diff
@@ -4,27 +4,27 @@ namespace Energy_Saver.Services
 {
     public class TableService : ITableService
```
Comment (priolette) [inline Energy Saver/Services/TableService.cs:5]: Weird way of deleting classes

File: Energy Saver/Services/SuggestionsService.cs (hunk 1 above)
Comment (Tauras-Rim) [reply Energy Saver/Services/SuggestionsService.cs:16]: There is already an if in place for that. this if will only be triggered if list.Count > 1

File: Energy Saver/Services/TableService.cs (hunk 3 above)
Comment (Tauras-Rim) [reply Energy Saver/Services/TableService.cs:5]: I wasn't sure if we would still need this so i only commented it out, we can fully delete it in develpment

File: Energy Saver/Model/Serialization.cs (hunk 2 above)
Comment (Tauras-Rim) [reply Energy Saver/Model/Serialization.cs]: Good point. I thought it was already done in a different branch. Changed it now

**Pull request merged**